tensorboard/plugins/histogram/histograms_plugin_test.py

```diff
@@ -173,6 +175,24 @@ def test_histograms_with_histogram(self):
     self._test_histograms(self._RUN_WITH_HISTOGRAM,
                           '%s/histogram_summary' % self._HISTOGRAM_TAG)
 
+  def _test_histograms_csv(self, run_name, tag_name, should_work=True):
```

Comment:
This isn't actually executed as a test, it would need to start with `test` (no underscore).  The code I referenced in the scalars plugin is a helper that's called by other test methods, sorry if that wasn't clear.

For this code, you would want to put this helper up next to the existing `_test_histograms()` renaming that one to `_test_histograms_json()` and then update the above `test_histograms_with_*` test methods to `test_histograms_json_with_*`, and then add three new copies that are `test_histograms_csv_with_*` that call this helper.

See the rest of the scalars test for what this should look like: https://github.com/tensorflow/tensorboard/blob/14512206d42af0ef0a0a1d72f3d78eaf4bae05b2/tensorboard/plugins/scalar/scalars_plugin_test.py#L193-L215